india_compliance/gst_india/overrides/purchase_invoice.py

```
@@ -53,10 +58,18 @@ def validate(doc, method=None):
     validate_reverse_charge(doc)
     validate_supplier_invoice_number(doc)
     validate_with_inward_supply(doc)
+    set_or_validate_itc_claim_period(doc)
     set_reconciliation_status(doc)
     set_pending_boe_qty(doc)
 
 
+def on_update_after_submit(doc, method=None):
```

Ruff (0.14.14)

Unused function argument: `method`

(ARG001)